doc/getting-started.md

```diff
@@ -0,0 +1,390 @@
+
+## Node configuration
+
+A server contain a lot of parameter to configure, from memory limits, to RPC credentials, ports... There is two ways to prepare your perfect node !
+
+Using directly `dogecoind` options, `-help` will display all configuration parameters.
+
+**Command example :**
+```bash
+dogecoind -daemon -paytxfee=0.01 -sendfreetransactions=1 -maxconnections=150
+```
+
+Otherwise, a node is configurable through a `dogecoin.conf` file. Create it in the directory defined with the `datadir` setting, `$HOME/.dogecoin` by default, or specify the file location with `-conf`.
+
+**dogecoin.conf example :**
+```
+daemon=1
+server=1
+listen=1
+paytxfee=0.01
+sendfreetransactions=1
+maxconnections=150
+```
+You can see a more concrete example [here](/contrib/debian/examples/dogecoin.conf).
+
+### Mainnet, testnet and regnet
+
+To perform tests, for example to create a blockchain based application, you may need a sandbox environment. Multiple networks exists for this purpose.
+
+**Mainnet** : The main network where real transaction operate.  
+**Testnet** : The test network, with peers.  
+**Regnet** : The regression network, to test without peers and create blocks on-demand.
+
+*Mainnet* is the network used by default. To enable *testnet*, with `dogecoind` use the `-testnet` option and `testnet=1` as `dogecoin.conf` field.
+
+To enable *regnet*, use `-regtest` option.
+
+> **Tips :** Think to specify the network when you use `dogecoin-cli`.
+
+### Data directory
+
+The data directory is the location where Dogecoin Core files are stored : `wallet.dat`, `debug.log`, blocks... You can modify it with the `-datadir` setting.
+
+**Default location :**
+
+Platform | Data directory path
+---------|--------------------
+Linux    | `$HOME/.bitcoin/`
+macOS    | `$HOME/Library/Application Support/Bitcoin/`
+Windows  | `%APPDATA%\Bitcoin\`
+
+You may need to specify `-datadir` also when using `dogecoin-cli`.
+
+See [full documentation on file system](/doc/files.md) for more information.
+
+### RPC credentials
+
+A username and a password is needed to control the RPC interface. When no credentials are provided, dogecoin use a [special cookie](https://bitcoin.org/en/release/v0.12.0#rpc-random-cookie-rpc-authentication) to randomly generate it. It's possible to define your own identifiers using `rpcuser` and `rpcpassword` settings.
+
+### Ports
+
+A node exposes up to 2 ports. One mandatory port for the **Peer to Peer Network (P2P)** to communicate with other nodes, a second optional port expose the RPC API.
+
+| Function | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+To configure them use `port` and `rpcport` settings.
+
+### Memory
+
+Dogecoin Core use important ressources for some environment, it may be interesting to restrict them. See [related documentation](/doc/reduce-memory.md) for information on how to reduce the memory footprint.
```

Comment:
```suggestion
Running Dogecoin Core can require a lot of memory, so in some situations it may be necessary to optimize its usage. You can find more information about reducing the memory footprint in the [related guide](reduce-memory.md).
```